app/controllers/BoughtOrSoldOverThresholdController.scala

```diff
@@ -65,7 +65,7 @@ class BoughtOrSoldOverThresholdController @Inject()(
         value =>
           for {
             updatedAnswers <- Future.fromTry(request.userAnswers.set(BoughtOrSoldOverThresholdPage, value))
-            _              <- sessionRepository.set(updatedAnswers)
+            _              <- sessionRepository.set(request.internalId, updatedAnswers)
```

Comment:
Can we please rename this to `credId' here;

/Users/chilts78/dev/amls-art-market-participant-frontend/app/models/requests/DataRequest.scala

Reply:
Both places?

Reply:
Done